index.bs

```diff
@@ -1,17 +1,16 @@
 <pre class='metadata'>
 Group: AOM
-Status: FD
-Text Macro: SPECVERSION v1.1.0
+Status: WGD
 Title: AV1 Image File Format (AVIF)
-URL: https://AOMediaCodec.github.io/av1-avif/v1.1.0.html
+URL: https://AOMediaCodec.github.io/av1-avif
 !Latest version: <a href="https://AOMediaCodec.github.io/av1-avif">https://AOMediaCodec.github.io/av1-avif</a>
-!Previously approved version: <a href="v1.0.0.html">v1.0.0</a>
+!Previously approved version: <a href="v1.1.0.html">v1.1.0</a>
```

Comment:
Can you align the headers with the ones in AV1-ISOBMFF https://aomediacodec.github.io/av1-isobmff/ 
```
This version:
https://aomediacodec.github.io/av1-avif/
Previously approved version:
https://aomediacodec.github.io/av1-avif/v1.1.0.html
Latest approved version:
https://aomediacodec.github.io/av1-avif/latest-approved.html
Latest draft version:
https://aomediacodec.github.io/av1-avif/latest-draft.html
Test Suite:
https://aomediacodec.github.io/av1-isobmff/conformance/
Issue Tracking:
GitHub
```
The latest-approved.html and latest-draft.html should be symlinks to the corresponding versions.